README.md

```diff
@@ -25,15 +26,19 @@ For more information, see the [Intel TDX overview](https://www.intel.com/content
 This tech preview of Intel TDX on Ubuntu 24.04 provides base host OS, guest OS, and remote attestation functionalities.
 Follow these instructions to setup the Intel TDX host, create a TD, boot the TD, and attest the integrity of the TD's execution environment.
 
-The setup can be customized by editing the global configuration file: `setup-tdx-config`
-
 <a id="report-an-issue"></a>
 ## 2. Report an Issue
-Please submit an issue [here](https://github.com/canonical/tdx/issues) and we'll get back to you ASAP.
+Please submit issues [here](https://github.com/canonical/tdx/issues) and we'll get back to you ASAP.
+
+To help us with the debugging process, run the [system-report.sh](https://github.com/canonical/tdx/blob/noble-24.04/system-report.sh)
```

Comment:
Requesting the user to run the system-report is already present when issue create an issue, i dont think it is necessary to repeat it here

Reply:
I think it does not hurt to mention something like this multiple times. Many customers will still not do it.

Reply:
Agree with @BFuhry.

Reply:
majority wins